src/Tribe/PUE/Checker.php

```diff
@@ -345,6 +350,10 @@ public function set_key_status( $valid ) {
 			// We set a transient in addition to an option for compatibility reasons.
 			// @todo remove transient in a major feature release where we release all plugins.
 			set_transient( $this->pue_key_status_transient_name, $status, $this->check_period * HOUR_IN_SECONDS );
+
+			if ( ! tribe_is_truthy( $valid ) ) {
+				delete_transient( self::IS_ANY_LICENSE_VALID_TRANSIENT_KEY );
+			}
```

Comment:
I think we care about being aware when a license is valid mostly not when invalid.

What is the downside if the transient is falsy saying there is one valid while there isn't ?

Also make sure you `set_transient( self::IS_ANY_LICENSE_VALID_TRANSIENT_KEY, $status, HOUR_IN_SECONDS );` when the license is valid.